ontology/yaml/resources/states/states.yaml

```diff
@@ -70,6 +70,19 @@ NOSOURCE: "An operation mode having no source as priority"
 POSITIONA: "Automatic Transfer switch position 0"
 POSITIONB: "Automatic Transfer switch position 1"
 POSITIONC: "Automatic Transfer switch position 2"
+
+ALARM: "Device is not in normal operation"
```

Comment:
This can be removed, since it is not used elsewhere, yes? Every other alarm follows ACTIVE/INACTIVE with this most recent change.